lib/transports/rest2.js

```diff
@@ -15,7 +15,12 @@ const {
   Trade,
   Wallet,
   Alert,
+<<<<<<< HEAD
   Tick
+=======
+  Ticker,
+  Candle
+>>>>>>> 03c086b... (refactor) improve REST2 methods that call through to REST1
```

Comment:
here is an unresolved conflict

Reply:
Fixed, it was a remnant of awaiting the rebase I think.